src/mixins/Regions.js

```diff
@@ -179,53 +121,6 @@ const RegionsMixin = types
         console.error('Region class needs to implement serialize');
       },
 
-      toStateJSON() {
```

Comment:
Just for clarity, can you explain the removal of `toStateJSON`?

Reply:
It's some kind of legacy code. We do not use it at all.